cli/klaus/src/logger.rs

```diff
@@ -0,0 +1,42 @@
+use std::fs::OpenOptions;
+use std::path::PathBuf;
+
+use anyhow::Context;
+use tracing_subscriber::layer::SubscriberExt;
+use tracing_subscriber::util::SubscriberInitExt;
+use tracing_subscriber::{EnvFilter, Layer};
+
+/// Initializes a logger that persists logs to an optional file in addition to `stderr`
```

Comment:
Sure, I'll add that later. For now, the answer is:
- `flox` doesn't log to a file at all
- `klaus` logs to a file and sets a default log level more talkative than `error`
- `klaus` doesn't hook into `tracing` to provide metrics. We'll probably want to include our metrics library at some point, but right now that lives in `flox` and would need to be extracted so that `flox` and `klaus` don't have a circular depenency.
- `klaus` sets up metrics once, whereas `flox` calls `init_logger` 3 different times, setting it with different configuration each time. `klaus` just does much less here.